content/docs/for-developers/tracking-events/event.md

```diff
@@ -447,6 +447,12 @@ Engagement events include open, click, spam report, unsubscribe, group unsubscri
    </tbody>
 </table>
 
+<call-out type="warning">
+
+Events like deferrals or bounces may or may not have an IP included in their post. Some examples where this will happen may include in internal deferrals. This is where we have already determined an issue at a specific MX records and are waiting for that issue to clear before trying to deliver more mail. Since no action is taken no IP can be logged tanking an action. Another example would be a delayed bounce. This is where mail is accepted for delivery but later is rejected after the SMTP conversation is over. Since the SMTP conversation is no longer happening a new conversation is started where much of the previouse context is lost. This results in delayed bounces not having an IP and other information.
```

Comment:
s/`tanking`/`taking`
s/`previouse`/`previous`
Do we use the term `delayed bounce` with customers?